Cargo.toml

```diff
@@ -35,3 +35,5 @@ regex = "1"
 glob = "0"
 notify = "4"
 shellexpand = "1"
+tokio = { version = "0.2.13", features = [ "rt-core", "rt-threaded", "macros", "blocking" ] }
```

Comment:
I'm not familiar with tokio, but do all the features are required here?

Reply:
Actually, I think we can ditch `rt-core`, as we have `rt-threaded` already. The other are two needed. Updated to remove that extra feature.